navigation/navigation-compose/src/androidMain/kotlin/androidx/navigation/compose/ComposeNavGraphNavigator.android.kt

```diff
@@ -51,5 +52,8 @@ internal actual class ComposeNavGraphNavigator actual constructor(
 
         internal actual var popExitTransition: (@JvmSuppressWildcards
         AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition?)? = null
+
+        internal actual var sizeTransform: (@JvmSuppressWildcards
+        AnimatedContentTransitionScope<NavBackStackEntry>.() -> SizeTransform?)? = null
```

Comment:
I wasn't sure if the usage of this is something that could be supported yet so I left it in.

Reply:
Looks like I was able to use it